contents/docs/self-host/deploy/aws.mdx

```diff
@@ -13,10 +13,12 @@ import UpgradingSnippet from './snippets/upgrading'
 import UninstallingSnippet from './snippets/uninstalling'
 import TryUnsecureSnippet from './snippets/tryunsecure'
 
-First, we need to set up a Kubernetes cluster (see the official AWS [documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) for more info).
+First, we need to set up a Kubernetes cluster (see the official AWS [documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) for more info). Follow the "Managed nodes - Linux" guide.  
 
 <ClusterRequirementsSnippet />
 
+Note that AWS by default has `allowVolumeExpansion` set to `false`, in the default EKS cluster this can be changed with `kubectl patch sc gp2 -p '{"allowVolumeExpansion": true}'`, see more info in [eks user guide for storage classes](https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html).
```

Comment:
Isn't this in the `ClusterRequirementsSnippet` snippet above? Here's the markdown content 👇 

* Suggestion: ensure `AllowVolumeExpansion` is set to `True` in the storage class definition (this setting enables `PVC` resize)
    <details>

    `PersistentVolumes` can be configured to be expandable. This feature when set to `true`, allows the users to resize the volume by editing the corresponding `PersistentVolumeClaims` object.

    This can become useful in case your storage usage grows and you want to resize the disk on-the-fly without having to resync data across PVCs.

    To verify if your storage class allows volume expansion you can run:

    ```shell
    kubectl get storageclass -o json | jq '.items[].allowVolumeExpansion'
    true
    ```

    In case it returns `false`, you can enable volume expansion capabilities for your storage class by running:

    ```shell
    kubectl patch storageclass -p '{"allowVolumeExpansion": true}'
    storageclass.storage.k8s.io/gp2 patched
    ```

    > N.B:
    >  - expanding a persistent volume is a time consuming operation
    >  - some platforms have a per-volume quota of one modification every 6 hours
    >  - not all the volume types support this feature. Please take a look at the [official docs](https://kubernetes.io/docs/concepts/storage/storage-classes/#allow-volume-expansion) for more info

    </details>

Reply:
Reading is hard 😅😅😅